### ISSUE TITLE
Login URL included in jetpackCarouselStrings variable

### ISSUE THREAD

**Issue by galbaras** [Automattic/jetpack#18773]
Google Search Console is pointing out URLs as "Indexed, though blocked by robots.txt". The only reason I can find for this is the inclusion of those same URLs in the script variable `jetpackCarouselStrings`.

Please remove this URL from carousel settings.

**Comment by jeherve** [comment]
That URL may be used by the comment form included in the Carousel; this is why we've added it there.

The link is not included in the markup by default though; it will only be there if your site is set to require registration for comments. 

Do you require user registration to comment on your site? If not, do you see the link in the markup of your page?

**Comment by galbaras** [comment]
No. "Users must be registered and logged in to comment" is unchecked in discussion settings.

**Comment by jeherve** [comment]
Could you post a link to your site so I can take a closer look at the markup?
Thank you!

**Comment by galbaras** [comment]
Sure. Look at https://www.ronitbaras.com/books/parenting-books/motivating-kids/

**Comment by jeherve** [comment]
Thanks! It's interesting that Google picks it up even though it's not used. It's something that we could improve, by only adding the string to the list of localized strings if necessary.

**Comment by galbaras** [comment]
It's a "yes" from me ;P
